app/src/wavevr/cpp/native-lib.cpp

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
 
   // Must initialize render runtime before all OpenGL code.
   WVR_RenderInitParams_t param;
-  param = { WVR_GraphicsApiType_OpenGL, WVR_RenderConfig_Timewarp_Asynchronous };
+  param = { WVR_GraphicsApiType_OpenGL, WVR_RenderConfig_Timewarp_Asynchronous | WVR_RenderConfig_MSAA};
```

Comment:
I'm confused, what is getting anti-aliased? Our FBOs already have 4x multi sampling. Does this cause the distortion pass to also multi-sample? I tried looking at the docs but it wasn't very clear to me.